learningobjects/migrations/0001_initial.py

```diff
@@ -0,0 +1,80 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+from django.conf import settings
+
+# auto-generated code, so shut up
+# pylint: disable=line-too-long,invalid-name,missing-docstring
```

Comment:
instead of adding this, I believe I suggested you fix it in the pylintrc, that way it will work for all migration files